js/layout-javascript/small-card-code.js

```diff
@@ -1743,6 +1743,11 @@ DynamicList.prototype.collapseElement = function(elementToCollapse) {
   }
 
   var collapseTarget = elementToCollapse.parent();
+
+  if (!collapseTarget || !collapseTarget.offset()) {
```

Comment:
If `elementToCollapse` was able to `.parent()`, it means that `elementToCollapse` is a jQuery object.

This means `collapseTarget` will always be a jQuery object too and truthy. You should check for `collapseTarget.length` instead. If you test it on jquery.com you'll find that `$('asdkhasdjsad').offset()` returns `undefined`, which means the problem here most likely comes from `collapseTarget` is an empty array.

```suggestion
  if (!collapseTarget.length) {
```

Reply:
Got it! Fixed